x-pack/solutions/security/plugins/cloud_defend/public/components/control_general_view_selector/index.test.tsx

```diff
@@ -667,12 +667,11 @@ describe('<ControlGeneralViewSelector />', () => {
     const selector = getByTestId('cloud-defend-selector');
 
     // should start as closed.
-    // there are two mock selectors, and the last one will auto open
```

Comment:
This was incorrect. The old length of `2` was not a result of 2 selectors being rendered but each `EuiAccordionTrigger` rendering two collapse/expand buttons - one button icon for the arrow and one as a wrapper for the `buttonContent`. This was recently fixed in https://github.com/elastic/eui/pull/8342 by adding `aria-hidden` to the button icon making it ignored by RTL